content/blog/2020-06-29-release-notes.md

```diff
@@ -68,7 +68,7 @@ export const EditLink = () => {
     <button
       onClick={
 -       github.editMode ? github.exitEditMode : github.enterEditMode
-+       cms.enabled ? cms.disable() : cms.enable()
++       cms.enabled ? cms.disable : cms.enable
```

Comment:
Good catch! They are, but that won't work. I mistakenly didn't make those functions lamdas. This might be better:

```suggestion
+       () => cms.toggle()
```

Reply:
[This PR will fix things](https://github.com/tinacms/tinacms/pull/1309)